types/mongodb/index.d.ts

```diff
@@ -172,6 +172,8 @@ export interface DbCreateOptions extends CommonOptions {
     // Sets a cap on how many operations the driver will buffer up before giving up on getting a
     // working connection, default is -1 which is unlimited.
     bufferMaxEntries?: number;
+    // The name of the application that created this MongoClient instance. 
+    appname?: string;
```

Comment:
`appname` should be added to the `MongoClientOptions` interface, not to `DbCreateOptions`

Reply:
Thanks for the feedback - this is updated.